src/utils/validateActionsJson.js

```diff
@@ -1,14 +1,14 @@
 import { schema } from 'taskcluster-lib-urls';
 import ajv from './ajv';
 
-let actionsJsonSchemaResponse = null;
-let validateActionsJson = null;
+let actionsJsonSchemaResponse;
+let validateActionsJson;
 
 export default async () => {
   if (!validateActionsJson) {
     actionsJsonSchemaResponse = await fetch(
       schema(
-        `https://${process.env.TASKCLUSTER_ROOT_URL}`,
+        process.env.TASKCLUSTER_ROOT_URL,
```

Comment:
Any reason to not use `urls.schema` here instead of re-using the environment variable?

Reply:
I should definitely use `urls.schema`. Thank you sir 💯